src/rhumb.js

```diff
@@ -114,13 +114,19 @@ function create (){
   }
 
   router.match = function(path){
-      var parts = path.split("/").filter(falsy)
-        , match  = findIn(parts, tree)
-
-      if(match){
-        return match.fn.apply(match.fn, [match.params])
+    
+    var split = path.split("?").filter(falsy)
+      , parts = split[0].split("/").filter(falsy)
+      , params = parseQueryString(split[1])
+      , match = findIn(parts, tree)
+
+    if(match){
+      for (var prop in match.params) {
```

Comment:
A bit of a nit, but `Object.keys` will avoid enumerating properties on the prototype chain, whereas `for..in` won't. It probably doesn't matter though.